cmd/podman/stats.go

```diff
@@ -89,15 +89,10 @@ func statsCmd(c *cli.Context) error {
 		times = 1
 	}
 
-	var format string
 	var ctrs []*libpod.Container
 	var containerFunc func() ([]*libpod.Container, error)
 
-	if c.IsSet("format") {
-		format = c.String("format")
-	} else {
-		format = genStatsFormat()
-	}
+	format := genStatsFormat(c.String("format"))
```

Comment:
format doesn't seem to be used until the for loop ~line 130.  Given there are a few error conditions in between, could you move this down nearer to that for loop?

Reply:
moved.